isce2_topsapp/localize_dem.py

```diff
@@ -42,7 +42,7 @@ def download_dem_for_isce2(extent: list,
                            dem_name: str = 'glo_30',
                            full_res_dem_dir: Path = None,
                            low_res_dem_dir: Path = None,
-                           buffer: float = .1) -> dict:
+                           buffer: float = .4) -> dict:
     """
     Parameters
     ----------
```

Comment:
Will want to update the docstring description for `buffer`.

Reply:
Also, this may be a good opportunity to note that the second `ymin` should be changed to `ymax` in the docstring description for `extent`.

Reply:
Thanks! I was looking at this code because ASF plans to utilize it as part of our [HyP3 ISCE2 plugin](https://github.com/ASFHyP3/hyp3-isce2).

Reply:
Please take a look at this too: https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/pull/122

This has some of the features that may be particularly relevant for on-demand processing including:

1. unfiltered coherence
2. 30 meter resolution

I think we are also discussing putting in Reference and Secondary Backscatter as additional layers. Hope this helps. Glad this is (potentially) being integrated as on-demand processing.  If the stars align maybe we can share some basic hyp3 tests or piggy back off your dependabot. The ISCE2 repo dependencies often give me a 🤯.

Reply:
Thanks, we'll take a look. @forrestfwilliams and I were just discussing how we might reduce duplication between the two projects so I will run all of this by him as well.

Reply:
Yes, definitely hope we can combine efforts!